### PR TITLE
Service portal/remove min rettindi

### PR DESCRIPTION
**Pull request by Thorrik58**
# Service portal/remove min rettindi feature flag

Attach a link to issue if relevant

## What

Remove min rettindi feature

## Why

Shouldnt be visible.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by obmagnusson — APPROVED** [review]
LGTM, Although delegation is a strange name for this.

File: apps/service-portal/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ export default {
   production: true,
   identityServer: {
     IDENTITY_SERVER_ISSUER_URL:
-      window.location.origin === 'https://beta.minarsidur.staging01.devland.is'
-        ? 'https://beta.minarsidur.dev01.devland.is'
+      window.location.origin === 'https://service-portal.staging01.devland.is'
```
Comment (Thorrik58) [inline apps/service-portal/src/environments/environment.prod.ts:5]:
```suggestion
      window.location.origin === 'https://beta.staging01.devland.is/minarsidur'
```

Comment (obmagnusson) [inline apps/service-portal/src/environments/environment.prod.ts:5]:
```suggestion
      window.location.origin === 'https://beta.staging01.devland.is'
```

File: apps/service-portal/src/environments/environment.prod.ts [hunk 2]
```diff
@@ -2,8 +2,8 @@ export default {
...
+        ? 'https://identity-server.staging01.devland.is'
```
Comment (Thorrik58) [inline apps/service-portal/src/environments/environment.prod.ts:6]:
```suggestion
        ? 'https://beta.dev01.devland.is/minarsidur'
```

File: apps/service-portal/src/environments/environment.prod.ts [hunk 3]
```diff
@@ -2,8 +2,8 @@ export default {
...
         : window.location.origin === 'https://beta.minarsidur.island.is'
```
Comment (obmagnusson) [inline apps/service-portal/src/environments/environment.prod.ts:7]:
```suggestion
        : window.location.origin === 'https://island.is'
```

File: apps/service-portal/src/environments/environment.prod.ts [hunk 4]
```diff
@@ -2,8 +2,8 @@ export default {
   production: true,
   identityServer: {
     IDENTITY_SERVER_ISSUER_URL:
-      window.location.origin === 'https://beta.minarsidur.staging01.devland.is'
-        ? 'https://beta.minarsidur.dev01.devland.is'
+      window.location.origin === 'https://beta.staging01.devland.is/minarsidur'
```
Comment (Thorrik58) [inline apps/service-portal/src/environments/environment.prod.ts:5]:
```suggestion
      window.location.origin === 'https://beta.staging01.devland.is'
```

File: apps/service-portal/src/environments/environment.prod.ts [hunk 5]
```diff
@@ -2,9 +2,9 @@ export default {
   production: true,
   identityServer: {
     IDENTITY_SERVER_ISSUER_URL:
-      window.location.origin === 'https://beta.minarsidur.staging01.devland.is'
-        ? 'https://beta.minarsidur.dev01.devland.is'
-        : window.location.origin === 'https://beta.minarsidur.island.is'
+      window.location.origin === 'https://beta.staging01.devland.is'
+        ? 'https://beta.dev01.devland.is/minarsidur'
```
Comment (Thorrik58) [inline apps/service-portal/src/environments/environment.prod.ts:6]:
```suggestion
        ? 'https://beta.dev01.devland.is'
```

File: apps/service-portal/src/environments/environment.prod.ts [hunk 6]
```diff
@@ -2,8 +2,8 @@ export default {
   production: true,
   identityServer: {
     IDENTITY_SERVER_ISSUER_URL:
-      window.location.origin === 'https://beta.minarsidur.staging01.devland.is'
-        ? 'https://beta.minarsidur.dev01.devland.is'
+      window.location.origin === 'https://beta.staging01.devland.is/minarsidur'
+        ? 'https://beta.dev01.devland.is/minarsidur'
```
Comment (obmagnusson) [inline apps/service-portal/src/environments/environment.prod.ts]:
```suggestion
        ? 'https://identity-server.staging01.devland.is/'
```

File: apps/service-portal/src/environments/environment.prod.ts [hunk 7]
```diff
@@ -2,9 +2,9 @@ export default {
   production: true,
   identityServer: {
     IDENTITY_SERVER_ISSUER_URL:
-      window.location.origin === 'https://beta.minarsidur.staging01.devland.is'
-        ? 'https://beta.minarsidur.dev01.devland.is'
-        : window.location.origin === 'https://beta.minarsidur.island.is'
+      window.location.origin === 'https://beta.staging01.devland.is'
+        ? 'https://beta.dev01.devland.is'
```
Comment (obmagnusson) [inline apps/service-portal/src/environments/environment.prod.ts:6]:
```suggestion
        ? 'https://identity-server.dev01.devland.is'
```

Comment (obmagnusson) [inline apps/service-portal/src/environments/environment.prod.ts:6]:
```suggestion
        ? 'https://identity-server.staging01.devland.is'
```

**Review by obmagnusson — APPROVED** [review]
LGTM

**Pull request merged**